lib/exercism/github_user.rb

```diff
@@ -0,0 +1,10 @@
+class GithubUser
```

Comment:
GitHub has a capitalized H. I think I got this wrong in the original module name, and it's been trailing along ever since.